docs/dev/developer-guides/building-on-zksync/tools.md

```diff
@@ -0,0 +1,17 @@
+# Choosing the right tool
+
+How do you use your favorite tools for building contracts when you're working with Optimism? Just use them!
+
+zkSync EVM compatible, which means all the Ethereum tooling you use, should work on zkSync without much stress.
+
+## Tools available on zkSync
+
+We are currently adding more tools to our belt, to enable developers to build with their favorite tools.
```

Comment:
We are currently adding more tools to our belt. Get in touch through our [Discord](https://discord.com/invite/px2aR7w) if you want to request specific ones.